packages/fuels-test-helpers/src/node.rs

```diff
@@ -43,6 +44,7 @@ impl Config {
             manual_blocks_enabled: false,
             vm_backtrace: false,
             silent: true,
+            poa_interval_millis: 2500,
```

Comment:
Note that this will be temporary while we wait for the bad default value in the `fuel-core` CLI to be fixed. Once fixed, we'll bring back the `Instant` config.

Reply:
Add a //TODO or open an issue?